platform/ios/src/MGLMapView.mm

```diff
@@ -750,6 +753,20 @@ - (void)glkView:(__unused GLKView *)view drawInRect:(__unused CGRect)rect
         _mbglMap->render();
 
         [self updateUserLocationAnnotationView];
+        
+        // Hide all annotation views
```

Comment:
This code runs 60 times a second due to the CADisplayLink. Intuitively, I’d expect hiding, repositioning, and showing views in this relatively hot code to be a bit of a drag on performance. Can we move this into a separate method that gets called when appropriate, for example in `-notifyMapChange:`, as well as in `-observeValueForKeyPath:ofObject:change:context:` when an annotation’s `coordinate` key path changes?